src/trainingdata/rescorer.h

```diff
@@ -27,13 +27,17 @@
 
 #pragma once
 
-#include <thread>
+#include <vector>
 
-#include "chess/uciloop.h"
-#include "utils/optionsparser.h"
+#include "syzygy/syzygy.h"
+#include "trainingdata/trainingdata_v6.h"
 
 namespace lczero {
 
 void RunRescorer();
+std::vector<V6TrainingData> RescoreTrainingData(
+    const std::vector<V6TrainingData>& fileContents, SyzygyTablebase* tablebase,
```

Comment:
I believe all updates of the training data are in-place.

If so, I believe it makes sense either to pass the vector by reference and make it update in-place, or

(probably easier) pass it by value so that it's moved into in the call and then moved back on return.

That would avoid an extra copy.